### DISCUSSION TITLE
Header and Footer size

### DISCUSSION THREAD

**Discussion by ivanbarayev** [acikkaynak/eczane-frontend#125] [Ideas]
From a user experience perspective, the header and footer size is too large (vertically), restricting the visibility and usage of the Map.

```
Tried resolutions (DESKTOP): 
- 1280x1024
- 1920x1200 
- 2560x1440
```

**Note** : The images belong to 2560x1440, but same UX issue in other resolutions 

![image](https://user-images.githubusercontent.com/6764405/218518925-3babc235-1ccf-43f1-911b-512ef5a7a7a0.png)

---

Elements in Header and Footer also need to be resized [Ref to : #69 ]

### Header
![image](https://user-images.githubusercontent.com/6764405/218519112-932defd3-0b57-4dab-9ba3-49d4f64d6ab7.png)

### Footer
![image](https://user-images.githubusercontent.com/6764405/218519356-06483617-1a86-45f6-a720-cab2127ac977.png)

Tried devices (iPhone 11-12-SE-XR) (safari and chrome):
Generally scrolldown issue (not enough space for touch area )

**Comment by htkarabulut** [comment]
Hi,
I think this issue already exists, check #69. Thank you so much for informing us though.

**Comment by ivanbarayev** [comment]
The main point is not same as issue #69, and i mentioned the issue in the description. 
Issue related with map and touch screen area on UX side.

**Comment by okazkayasi** [comment]
Reopened. Delivered the feedback to design team. Will notify with upcoming progress @ivanbarayev
